<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input</class>
 <widget class="QWidget" name="Input">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>321</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CBNU Grading Management System</string>
  </property>
  <widget class="QPushButton" name="pushButton_input">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>270</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>199</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_year">
        <property name="text">
         <string>연도</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_semester">
        <property name="text">
         <string>학기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_division">
        <property name="text">
         <string>이수구분</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lecture">
        <property name="text">
         <string>교과목명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_credit">
        <property name="text">
         <string>학점</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_grade">
        <property name="text">
         <string>등급</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_name"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_year"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_semester"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_category"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_subject"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_credit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_grade"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>94</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_isvalid">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>236</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>유효성 검사</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_validate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_input</zorder>
  <zorder>pushButton_isvalid</zorder>
  <zorder>lineEdit_validate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
